### ISSUE TITLE
Nav overlay preview toggle button not perceiveable by assistive technology

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#53461]
As per feedback in https://www.youtube.com/watch?v=6H15BUb6_GU&t=3355s by @alexstine the button which toggles the display controls for the Navigation overlay is not accessible.

We need to make the control perceiveable.
